2.3.0: pytest fails with missing win32_setctime on unix oses

FreeBSD / Python 3.11

I've ported `knowit` and `trakit` (not committed yet) and subliminal is using following test dependencies:

```
TEST_DEPENDS=   ${PYTHON_PKGNAMEPREFIX}defusedxml>0:devel/py-defusedxml@${PY_FLAVOR} \
                ${PYTHON_PKGNAMEPREFIX}knowit>0:multimedia/py-knowit@${PY_FLAVOR} \
                ${PYTHON_PKGNAMEPREFIX}sympy>0:math/py-sympy@${PY_FLAVOR} \
                ${PYTHON_PKGNAMEPREFIX}tomlkit>0:textproc/py-tomlkit@${PY_FLAVOR} \
                ${PYTHON_PKGNAMEPREFIX}vcrpy>0:devel/py-vcrpy@${PY_FLAVOR}

```
 
I'm in doubt if I need to port `win32_setctime` too as it seems to be windows related?
I there a quick workaround to avoid it?

Thanks

```
platform freebsd15 -- Python 3.11.12, pytest-8.1.1, pluggy-1.5.0 -- /usr/local/bin/python3.11
cachedir: .pytest_cache
rootdir: /wrkdirs/usr/ports/multimedia/py-subliminal/work-py311/subliminal-2.3.0
configfile: pyproject.toml
collected 590 items / 1 error / 7 deselected / 583 selected

====================================================================================================== ERRORS ======================================================================================================
_______________________________________________________________________________________ ERROR collecting tests/test_video.py _______________________________________________________________________________________
ImportError while importing test module '/wrkdirs/usr/ports/multimedia/py-subliminal/work-py311/subliminal-2.3.0/tests/test_video.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_video.py:10: in <module>
    from win32_setctime import SUPPORTED, setctime
E   ModuleNotFoundError: No module named 'win32_setctime'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================================================== 7 deselected, 1 error in 1.47s ==========================================================================================
*** Error code 2
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -73,7 +73,7 @@ tests = [
     "sympy",
     "colorama",  # used in scripts/
     "vcrpy>=5",  # keep synchronized with docs dependencies
-    "win32-setctime",
+    "win32-setctime; sys_platform=='win32'",
     "importlib_metadata>=4.6; python_version<'3.10'",
 ]
 types = [

--- a/tests/test_video.py
+++ b/tests/test_video.py
@@ -7,7 +7,6 @@ from typing import TYPE_CHECKING
 from unittest.mock import Mock
 
 import pytest
-from win32_setctime import SUPPORTED, setctime
 
 from subliminal.utils import sanitize, timestamp
 from subliminal.video import Episode, Movie, Video
@@ -40,6 +39,12 @@ def test_video_exists_age_no_use_ctime(
 
 
 def test_video_exists_age(movies: dict[str, Movie], tmp_path: Path, monkeypatch: pytest.MonkeyPatch) -> None:
+    try:
+        from win32_setctime import SUPPORTED, setctime
+    except (ImportError, ModuleNotFoundError):
+        SUPPORTED = False
+        setctime = None
+
     monkeypatch.chdir(tmp_path)
     video = movies['man_of_steel']
     video_path = tmp_path / video.name

